toast-tk-runtime/src/main/java/io/toast/tk/runtime/core/parse/WebPageSetupBlockParser.java

```diff
@@ -67,7 +76,7 @@ private static void assertIsTitle(final String firstLine) {
 
 	private static void assertLineHasEnoughColumn(String firstLine, String[] split) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "firstLine". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)